docs/UserGuide.md

```diff
@@ -5,40 +5,38 @@ title: MediBase3 User Guide
 
 MediBase3 (MB3) is a **desktop app for doctors to manage their patients and appointments, optimized for use via a Command Line Interface** (CLI) while still having the benefits of a Graphical User Interface (GUI). If you can type fast, MB3 can get your tasks done faster than traditional GUI apps.
 
+## Table of Contents
+
 * Table of Contents
 {:toc}
 
---------------------------------------------------------------------------------------------------------------------
-
 ## Quick start
 
 1. Ensure you have Java `17` or above installed in your Computer.
-
 1. Download the latest `.jar` file from [here](https://github.com/se-edu/addressbook-level3/releases).
-
 1. Copy the file to the folder you want to use as the _home folder_ for your AddressBook.
+1. Open a command terminal, `cd` into the folder you put the jar file in, and use the `java -jar addressbook.jar` 
+   command to run the application.asd
 
-1. Open a command terminal, `cd` into the folder you put the jar file in, and use the `java -jar addressbook.jar` command to run the application.<br>
-   A GUI similar to the below should appear in a few seconds. Note how the app contains some sample data.<br>
-   ![Ui](images/Ui.png)
-
-1. Type the command in the command box and press Enter to execute it. e.g. typing **`help`** and pressing Enter will open the help window.<br>
-   Some example commands you can try:
-
-   * `list` : Lists all contacts.
-
-   * `add n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01` : Adds a contact named `John Doe` to the Address Book.
-
-   * `delete 3` : Deletes the 3rd contact shown in the current list.
+    {: .alert .alert-secondary}
+    > A GUI similar to the below should appear in a few seconds. Note how the app contains some sample data.
+    > 
+    > ![Ui](images/Ui.png)
 
-   * `clear` : Deletes all contacts.
+1. Type the command in the command box and press Enter to execute it. e.g. typing **`help`** and pressing Enter will 
+   open the help window.
 
-   * `exit` : Exits the app.
+    {: .alert .alert-secondary}
+   > Some example commands you can try:
+   > * `list` : Lists all contacts.
+   > * `add n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01` : Adds a contact named `John Doe` tothe Address Book.
```

Comment:
should be to the not 'tothe'

Reply:
should not be address book. should be MediBase3

Reply:
I think its fine to leave it as that since, @TechRaj will settle the explanation